release-team/role-handbooks/release-team-lead/README.md

```diff
@@ -122,6 +122,26 @@ Release Team selection should happen in accordance with the [Release Team select
   - Kubernetes 1.16: Unlimited Breadsticks For All
   - Kubernetes 1.17: The Chillest Release - Capynetes
 
+### Mailing List Standards
+
+Important communications and discussions should be posted in the release team
+mailing list https://groups.google.com/forum/#!forum/kubernetes-release-team.
```

Comment:
This should be a pretty link instead. There's probably already one at the bottom of the doc for this.

Reply:
There is one at the bottom: `[kubernetes-release-team]`